Don't list target files that are directly added in a target (.target) while adding

vik-chand opened the pull request:
Don't list target files that are directly added in a target (.target) while adding (fixes #159)
Change-Id: If2edf9fbdfc1961b9379936936ad5886e1a05af3
Signed-off-by: Vikas Chandra <Vikas.Chandra@in.ibm.com>

akurtakov commented on Jenkinsfile:
```diff
@@ -19,7 +19,7 @@ pipeline {
 					mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 						-Pbuild-individual-bundles -Pbree-libs -Papi-check \
-						-Dcompare-version-with-baselines.skip=false \
+						-Dcompare-version-with-baselines.skip=true \
```
This change will make verification build not fail when version bumps are needed thus it sholdn't happen.

vik-chand replied:
Yes I was trying out this as I still had the " [ERROR] META-INF/ECLIPSE_.SF: present in baseline only". The intention was to check for other issues and then revert in the next commit  in this PR( as already done)

akurtakov commented:
I would suggest keeping the first commit only and rebase it on master to see real test results.

akurtakov commented:
Please rebase your branch on master.

vik-chand commented:
I have rebased on master. When I removed baseline check, mostly unrelated API tool errors were coming.. The current change has only 1 file ( which is required). I will squash and merge this one  ( will track in next I build) and hope I dont end up with "META-INF/ECLIPSE_.RSA: present in baseline only" in the next PR.

akurtakov commented:
You haven't rebased or at least haven't pushed the rebase . Look at https://github.com/vik-chand/eclipse.pde/tree/issue-159 - "This branch is [5 commits ahead](https://github.com/vik-chand/eclipse.pde/compare/eclipse-pde:master...issue-159), [20 commits behind](https://github.com/vik-chand/eclipse.pde/compare/issue-159...eclipse-pde:master) eclipse-pde:master."

vik-chand commented:
Revision 2167cbde7f2404f57fa0a60bd50bb89ef95115bb - just the comment change

Revision 81328a4e70306ff4cde1778287329ae9e5f4369d  - with the fix


Both have 349 test failures. In 1st revision there is no change except the comment.
In the 2nd revision, there is the fix and still the same 349 failures.

So the failures are independent.
